packages/core/backend/sirius-components-collaborative/src/main/java/org/eclipse/sirius/components/collaborative/editingcontext/EditingContextEventProcessorRegistry.java

```diff
@@ -99,6 +106,8 @@ public synchronized Optional<IEditingContextEventProcessor> getOrCreateEditingCo
                         }
                     });
 
+                    this.editingContextEventProcessorEditingContextProcessors.forEach(eCEPECP -> eCEPECP.postProcess(editingContext));
```

Comment:
I'm not sure we need a postProcess but at the very least I'll move the code a couple lines below since the entire thing in not fully initialized since it's not accessible in the list of editing context event processors of the registry.